modules/UI/Sources/Theme/Colors/ColorName.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ColorName.swift
+//  UI
+//
+//  Copyright 2022 Thoughtworks, Inc. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+public enum ColorName: String {
```

Comment:
How about `Colors`, follow `Dimensions`

Reply:
Or we can call it `ColorPalette`, I prefer `Colors`

Reply:
Ok! I would like to adopt `Colors`